lib/wanda_web/controllers/fallback_controller.ex

```diff
@@ -24,6 +24,30 @@ defmodule WandaWeb.FallbackController do
     |> render(:"403")
   end
 
+  def call(conn, {:error, :check_not_found}) do
+    conn
+    |> put_status(:not_found)
+    |> put_view(json: ErrorJSON)
+    |> render(:"404", reason: "Referenced check was not found.")
+  end
+
+  def call(conn, {:error, :check_not_customizable}) do
+    conn
+    |> put_status(:forbidden)
+    |> put_view(json: ErrorJSON)
+    |> render(:"403", reason: "Referenced check is not customizable.")
```

Comment:
question: shouldn't we put `400` here?
it looks more a thing that the user did wrong, rather than don't having access to this change.
they will have access only if the check itself is modified. he cannot gain real access otherwise

Reply:
If someone calls directly the endpoint attempting to customize an existing check, but that check is not customizable I intend that as a forbidden operation.

Now, details. If you have a strong preference for a bad request I don't mind.

Reply:
Yeah, nevermind. It is a detail indeed.